src/System.Linq.Expressions/src/System/Linq/Expressions/ExpressionStringBuilder.cs

```diff
@@ -646,6 +646,8 @@ protected internal override Expression VisitUnary(UnaryExpression node)
                     Out("+");
                     break;
                 case ExpressionType.Quote:
+                case ExpressionType.PostIncrementAssign:
+                case ExpressionType.PostDecrementAssign:
```

Comment:
Note I'd likely be better to revisit this method a bit further, and eliminate the `default` case at the bottom by handling all cases explicitly. Just like the printing for binary expressions, the `default` case can then throw an exception for an unexpected node.